# 单个多层级体素信息
geometry_msgs/Point position  # 体素中心位置
uint8 layer                   # 层级类型: 0=BLOCK, 1=VOXEL, 2=SUBVOXEL